docs/api/inputs/run_button.md

```diff
@@ -0,0 +1,31 @@
+# Run Button
+
+```{eval-rst}
+.. marimo-embed::
+    @app.cell
+    def __():
+        b = mo.ui.run_button()
+        b
+        return
+
+    @app.cell
+    def __():
+        s = mo.ui.slider(1, 10)
+        s
+        return
+
+
+    @app.cell
+    def __():
+        mo.stop(b.value, "Click `run` to submit the slider's value")
```

Comment:
should this be `mo.stop(not b.value)`.

not sure if this hides to much, but could put a stop on the button too?

```python
b.stop(fallback=None)

// Run thing
```

or maybe `b.gate()`, `b.clicked()`

Reply:
Oops yes it should.

Something like that could be nice — maybe `b.gate()` ... might add in a follow-up